<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Генератор полосатых флагов</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QPushButton" name="generateButton">
                                <property name="text">
                                    <string>Сгенерировать флаг</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="saveButton">
                                <property name="text">
                                    <string>Сохранить флаг</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QLabel" name="infoLabel">
                                <property name="text">
                                    <string>Количество полос: не задано</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QLabel" name="flagLabel">
                        <property name="frameShape">
                            <enum>QFrame::Box</enum>
                        </property>
                        <property name="text">
                            <string>Нажмите "Сгенерировать флаг" для создания флага</string>
                        </property>
                        <property name="alignment">
                            <set>Qt::AlignCenter</set>
                        </property>
                        <property name="wordWrap">
                            <bool>true</bool>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>